ProjectSidewalk/SidewalkWebpage
# 2889 add user id to attribute api

Resolves #2889 

Adds User Id as a property returned from the `attributesWithLabels` API and a list of user ids to the `attributes` API. In both of these additions, the corresponding user id(s) are found by querying the `audit_task` table.

##### Before/After screenshots (if applicable)
`attributesWithLabels` API Before:
<img width="1052" alt="Screen Shot 2022-10-02 at 10 12 00 PM" src="https://user-images.githubusercontent.com/50668250/193505648-13335b35-2101-49e6-8e35-af67545e73c6.png">

`attributesWithLabels` API After:
![image](https://user-images.githubusercontent.com/50668250/193505511-d20c7206-ec98-4fb0-8c4c-8bc3c31c6ad8.png)

`attributes` API Before:
<img width="1035" alt="Screen Shot 2022-10-02 at 6 41 02 PM" src="https://user-images.githubusercontent.com/50668250/193505606-f1619975-406b-4ba6-981d-046f43fe4459.png">

`attributes` API After:
![image](https://user-images.githubusercontent.com/50668250/193505525-836c12aa-8c2a-4830-a298-db9bff06689d.png)


##### Testing instructions
1. Navigate to the API page.
2. Use the example API calls for `attributesWithLabels` and `attributes` to confirm that the returned data contains a user id and a list of user ids, respectively. Also try editing the parameters of the calls for other test scenarios.
3. Requesting and displaying the data visually (through a shapefile), is especially useful when comparing labels' user ids to attributes' user id lists.
